python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -529,15 +538,17 @@ def __init__(cls, name, bases, this_dict):
 
         # all Ganga classes must have (even empty) schema
         if cls._schema is None:
-            cls._schema = Schema.Schema(None, None)
+            cls._schema = Schema(None, None)
 
         this_schema = cls._schema
 
+        cls.__slots__ = ('__dict__', '_proxyObject')
```

Comment:
Yes but we still have to have it in the do_not_copy... frankly we should advertise some knowledge in the Proxy of what it will dynamically create if we want to try and make the construction more sane.

I think I saw some performance improvement in memory allocation speeds when using slots and a large (Order 100k+) objects being created which is almost quite common.